src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -89,6 +96,11 @@ ProfileInfoTable.propTypes = {
   data: PropTypes.array.isRequired,
   dataTransformer: PropTypes.func,
   className: PropTypes.string,
+  /**
+   * When `list` is truthy, additional a11y markup will be applied to the
+   * rendered table to treat it like a list
+   */
+  list: PropTypes.bool,
```

Comment:
I'm wondering if `list` is the clearest name because it seems like this is always something like a list. I'm not sure where the distinction is between when `list` should be true vs. false - is there a name that could be used to help describe that distinction?

Reply:
I'm not sure of the best attribute name to use for this. Yeah it's always something like a list. But the military info section, where each row is the exact same kind of info, is more like a list than the account security section where each row represents a very different piece of info. That's why Jen wanted different markup (adding `role="list"` and `role="listitem"` to the appropriate elements) for the Military Info section. I'm guessing that the Connected Apps section's table will also be treated like list, while the Direct Deposit section's table will be treated like the Account Security section.

As we build out the other sections we might find _more_ instances where the table markup needs to be different still. But I really hope not!

Reply:
Ah okay, that makes sense. So it would be used if each row is the same type of data, versus having different data types for each row that fit conceptually under the same roof... Hm 🤔 that is a tricky one

Reply:
Yeah, it was kind of amazing to see Jen so quickly digest the two different tables she looked at and break their markup down into what seemed the most semantic. It's very much a skill I haven't developed. I was just looking at the tables from the visual point of view, in which case they are effectively identical.